### PR TITLE
New Package: Ruby rubocop

### PR DESCRIPTION
**Pull request by stacyharper**

### CODE REVIEW

**Comment by Eluminae** [comment]
```
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received
```

Is this a common problem ? Should we just restart the build ? It seems to works on my machine.

File: srcpkgs/ruby-jaro_winkler/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'ruby-jaro_winkler'
+pkgname=ruby-jaro_winkler
+version=1.5.4
+revision=1
+archs=noarch
```
Comment (Chocimier) [inline srcpkgs/ruby-jaro_winkler/template]: Contains compiled part so isn't noarch.

**Comment by Eluminae** [comment]
How could I restart the travis builds ?

**Comment by Eluminae** [comment]
Yata! It is now mergeable.

**Comment by Eluminae** [comment]
up ! :kiss:

**Comment by Chocimier** [comment]
Does not run.
```
$ rubocop
Could not find 'ast' (>= 2.4.0) among 51 total gem(s)
```

Are /usr/bin/setup, /usr/bin/console necessary?

Should be rather named rubocop as mainly commandline utility.

**Comment by Eluminae** [comment]
I added ast but I got a new issue with progressbar. I'm on it

**Comment by Eluminae** [comment]
> Are /usr/bin/setup, /usr/bin/console necessary?

What do you mean?

**Comment by Chocimier** [comment]
> removed them with the post_install method

Good.

Still don't run: Could not find 'ruby-progressbar' (>= 1.7) among 52 total gem(s) (Gem::MissingSpecError).

Please fix rubocop's commit title.

**Comment by Eluminae** [comment]
> Still don't run: Could not find 'ruby-progressbar' (>= 1.7) among 52 total gem(s) (Gem::MissingSpecError).

Yes I'm on it. I dunno what is the problem atm

> Please fix rubocop's commit title.

Good catch, done it

**Comment by ericonr** [comment]
Ping

**Comment by abenson** [comment]
Seemingly abandoned.

**Pull request closed without merging**